CMakeLists.txt

```diff
@@ -306,6 +306,8 @@ set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${GTEST_INCLUDE_DIRS})
 find_package (VJSON REQUIRED)
 set (INCLUDE_DIRECTORIES ${INCLUDE_DIRECTORIES} ${VJSON_INCLUDE_DIRS})
 
+find_package (LibArchive REQUIRED)
```

Comment:
One more thing: shouldn't that come together with a FindLibArchive cmake module?

Reply:
Added! :)

Reply:
Moreover, I confirm that on SLE11 and on my machine it actually pick the correct libarchive.a from external install.